activesupport/test/fixtures/some_module.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module SomeModule
+  extend ActiveSupport::Concern
+
+  included do
+    # shouldn't raise
```

Comment:
This is misleading. I would adjust the comment to say `# shouldn't raise when module is required more than once`. I would also consider renaming the module to `SomeConcern`

Reply:
Happy with the renaming and I'll expand the comment tho, as above, I think it needs to be `when module loaded more than once` rather than `...required...`.